quic/s2n-quic-core/src/event/generated.rs

```diff
@@ -331,7 +331,7 @@ pub mod api {
     }
     #[derive(Clone, Debug)]
     #[non_exhaustive]
-    pub enum Frame {
+    pub enum Frame<'a> {
```

Comment:
This feels likely to be a breaking change? Previously callers were free to store the clone and store Frames if they wanted to (or just keep Frame in a struct, etc).

Reply:
Thanks for pointing that out. I encountered that too since s2n-quic integration tests record these Frames. I'll make a separate event for emitting this data instead